### PR TITLE
Added Add-Unit button to sticky header

### PR DESCRIPTION
**Pull request by scudmarx**
Add3

### CODE REVIEW

**Comment by BaptisteD** [comment]
It doesn't seem to work yet on my side. 

https://user-images.githubusercontent.com/2566847/146939631-e5cfaf08-791f-4cb4-8f7e-852582796967.mov

**Comment by scudmarx** [comment]
> It doesn't seem to work yet on my side.

Thanks, should be better now.

**Comment by BaptisteD** [comment]
Unfortunately it's not yet there. 

https://user-images.githubusercontent.com/2566847/147156653-ad7046d0-7b18-456a-9da8-609a23626427.mov

Also, the spacing between the unit name and the button increased, there is now a padding-top of 52px, I don't know why. And therefore the button is over the weapon's table. 

Capture d’écran 2021-12-22 à 22 20 08

**Comment by scudmarx** [comment]
Ugh, silly MUI bug. Should be fixed now.  He said.  Again.

File: styles/globals.css [hunk 1]
```diff
@@ -49,25 +49,31 @@ div[role="tooltip"] {
 }
 
 .sticky {
-  position: sticky;
+  position: sticky !important;
   top: 0;
   background-color: #FAFAFA;
   box-shadow: none !important;
 }
+.sticky.sticky-2 {
```
Comment (AdamLay) [inline styles/globals.css:57]: Would this be better as an inline style - seems like a very specific class? Thanks,

**Comment by AdamLay** [comment]
I've moved the button into the upgrade panel header, and changed about the padding a little. Will merge for now but @BaptisteD will likely have something to say on the padding, I can't get it to line up nicely lol!

**Comment by BaptisteD** [comment]
Haha, it's alright. 
Could you add a margin-bottom of 0.5rem to the button when in scrolled state ? 
It will breath a bit more and reduce the impression of misalignment with the other column.

**Comment by BaptisteD** [comment]
I'm wondering if we shouldn't add this previsualisation behavior on mobile to. 
It would require to remove the accordeon behavior in compact mode. But with the two modes available, I don't think it's really useful anymore. 

What do you think ?

**Pull request merged**